lib/perl/Genome/Config/AnalysisProject/Command/DiskUsage.pm

```diff
@@ -0,0 +1,102 @@
+package Genome::Config::AnalysisProject::Command::DiskUsage;
+
+use strict;
+use warnings;
+
+use Genome;
+
+class Genome::Config::AnalysisProject::Command::DiskUsage {
+    is => 'Command::V2',
+    has => [
+        analysis_projects => {
+            is                  => 'Genome::Config::AnalysisProject',
+            doc                 => 'the analysis projects to report disk usage',
+            is_many             => 1,
+            shell_args_position => 1,
+        },
+    ],
+    doc => 'report on the disk usage of analysis projects',
+};
+
+sub help_detail {
+    return <<EOHELP
+This command will take several minutes to run.
+
+The output is a tab-delimited disk usage summary for each model type and configuration item per analysis project.
+The colums of the output file are:
+analysis project ID, configuration item ID, subclass name, model count, build count, total KB, total base pair, bytes per base
+
+EOHELP
+}
+
+sub execute {
+    my $self = shift;
+
+    my @lines;
+    for my $anp ($self->analysis_projects) {
+        for my $config_item ($anp->config_items) {
+            my %model_type_disk_usage;
+            for my $model ($config_item->models) {
+                my $subclass_name = $model->subclass_name;
+                $model_type_disk_usage{$subclass_name}{'model_count'}++;
+                for my $build ($model->builds) {
+                #my $build = $model->last_succeeded_build;
+                #if ($build) {
```

Comment:
This commented-out code should probably be removed (unless we want to make this an option for the command?)